acceptance/go.mod

```diff
@@ -2,7 +2,10 @@ module github.com/hashicorp/consul-k8s/acceptance
 
 go 1.20
 
+replace github.com/hashicorp/consul/sdk => github.com/hashicorp/consul/sdk v0.4.1-0.20231130173447-f44a9e1729df
```

Comment:
This points to my branch where I extended the Retry struct. When https://github.com/hashicorp/consul/pull/19813 gets merged.